packages/react-heartwood-components/src/components/List/components/PermissionsListItem/PermissionsListItem.js

```diff
@@ -0,0 +1,77 @@
+// @flow
```

Comment:
I don't think this should be in workspace since no other skills are going to have access. instead, move to web.

thoughts @gcpantazis @jtini

Reply:
I see what you mean, but if we keep it here, we can include it in Storybook.

Reply:
@liquidg3 updated to incorporate only the updates to `<List>`

Reply:
this is good, ready to review web ai when it's ready @CarolynSM